### ISSUE TITLE
TLS CRL optional argument 

### ISSUE THREAD

**Issue by volga629-1** [SIPp/sipp#736]
Hello Everyone, 
Small clarification  -tls_crl  should be optional flag ?

Version

```
[volga629@canldev01-fedora39 ~]$ sipp -v

 SIPp v3.7.2-TLS-SCTP-PCAP.

```

```
  -tls_crl         : Set the name for Certificate Revocation List file. If not specified, X509 CRL
                      is not activated.

```

Run task 

```
 sipp -t ln -sf run.xml -log_file /tmp/sipp.log -tls_cert cert.crt -tls_key private.key -tls_ca root.crt  target IP
2024-07-21	14:24:44.603074	1721586284.603074: TLS_init_context: Unable to load CRL file ()
2024-07-21	14:24:44.603074	1721586284.603074: TLS_init_context: Unable to load CRL file ()

**Comment by volga629-1** [comment]
Scanario basic UAC

**Comment by orgads** [comment]
Dup of #663. Fixed in 3.7.3.
